src/main/java/teammates/ui/pagedata/AdminActivityLogPageData.java

```diff
@@ -77,10 +78,21 @@ private void setDefaultLogSearchPeriod() {
     public void init(boolean ifShowAll, boolean ifShowTestData, List<ActivityLogEntry> logs) {
         this.ifShowAll = ifShowAll;
         this.ifShowTestData = ifShowTestData;
-        this.logs = logs;
+        initLogAsTemplateRow(logs);
         
     }
     
+    private void initLogAsTemplateRow(List<ActivityLogEntry> entries) {
+        this.logs = new ArrayList<AdminActivityLogTableRow>();
+        for (ActivityLogEntry entry : entries) {
+            AdminActivityLogTableRow row = new AdminActivityLogTableRow(entry);
+            this.logs.add(row);
+            if (q != null && q.isInfoInQuery) {
+                row.setKeyStringsToHighlight(q.infoValues);
+            }
```

Comment:
It should not make a difference functionally, but I feel like the flow is better if this if block comes before `this.logs.add(row)`.